storage/Filesystems/MountPoint.h

```diff
@@ -108,8 +108,15 @@ namespace storage
 	void set_mount_options(const std::vector<std::string>& mount_options);
 
 	/**
-	 * Set the mount options to the default mount options. So far the
-	 * default mount options only contain the subvol for btrfs subvolumes.
+	 * Default mount options. So far the default mount options only contain
+	 * the subvol for btrfs subvolumes.
+	 */
+	std::vector<std::string> default_mount_options() const;
```

Comment:
This could be called get_default_mount_options for consistency.

Reply:
But I fear that having a method called `set_default_mount_options` and another called `get_default_mount_options` would give a false impression about how they work.

They would look as setter/getter for the same "attribute".

To be honest, I think that `MountableImpl#get_default_mount_options` should not include the `get_` prefix, it makes it look like a getter.

Reply:
Just to clarify, I'm not opposing to change the name to `get_default_mount_options` (in the ruby wrapper it will look the same with or without the `get_`). I just explaining that I intentionally broke consistency with the internal names on purpose, because I think the result is less confusing.
As mentioned, in the yast-storage-ng side, it will make no difference.

Reply:
Then for consistency rename the internal function.